execution-engine/src/native_modules/aes.rs

```diff
@@ -102,10 +102,7 @@ impl AesCounterModeService {
             let block_size = cypher.block_size();
             // For some reason, the length of the output buffer needs to be
```

Comment:
I know this isn't your comment, but can we find a resource describing why the output buffer needs to be padded as such and update this comment?